pwa/src/styling/design-tokens/component-overrides.css

```diff
@@ -280,3 +281,50 @@
 .denhaag-button:hover {
   cursor: pointer;
 }
+
+.denhaag-icon {
+  /* Den Haag has `font-size: 1.5rem` as hardcoded value, we can't work with that */
+  font-size: inherit !important;
+  /* Use the width and height from the parent `.utrecht-icon` class name */
+  width: var(--utrecht-icon-size, auto) !important;
+  height: var(--utrecht-icon-size, auto) !important;
+}
+
+.utrecht-icon svg {
+  /* Override the width of SVGs when they are hardcoded.
+   * For example:
+   * <svg width="128px" height="128px">
+   */
+  height: 100%;
+  width: 100%;
+}
+
+/*
+ * For icons that are placed at start, before a text label.
+ * Example: previous arrow
+ */
+.utrecht-icon--conduction-start {
+  margin-inline-end: var(--conduction-icon-gap);
```

Comment:
See other comment.

This would become `margin-inline-end: var(--web-app-icon-gap)`.

Also, I think the name should be more specific, e.g. `--web-app-link-icon-gap`.